clickhouse_driver/result.py

```diff
@@ -113,3 +114,12 @@ def next(self):
 
     # For Python 3.
     __next__ = next
+
+
+class QueryInfo(object):
+    def __init__(self):
+        self.profile_info = None
+
+    def store(self, packet):
+        if packet.type == ServerPacketTypes.PROFILE_INFO:
```

Comment:
We are already checked for packet type in `client.py`
```python
        elif packet.type == ServerPacketTypes.PROFILE_INFO:
            self.last_query.store(packet)
```

Reply:
Yeah, I was actually wondering if there might be some other stuff worth saving, but for now I've simplified this.